core/src/main/scala/org/apache/spark/sql/rf/QuinaryExpression.scala

```diff
@@ -0,0 +1,111 @@
+package org.apache.spark.sql.rf
+
+import org.apache.spark.sql.catalyst.expressions.codegen.Block._
+import org.apache.spark.sql.catalyst.InternalRow
+import org.apache.spark.sql.catalyst.expressions.Expression
+import org.apache.spark.sql.catalyst.expressions.codegen.{CodeGenerator, CodegenContext, ExprCode, FalseLiteral}
+
+/**
+ * An expression with five inputs and one output. The output is by default evaluated to null if any input is evaluated to null
+ */
+abstract class QuinaryExpression extends Expression {
```

Comment:
There is only `Quaternary` and `Septenary` in the spark core.

Reply:
I think those might be new! I think it topped out at trinary before?

Reply:
@metasim I think you're 💯 correct